building/build-debs/homeworld-admin-tools/inner-build.sh

```diff
@@ -3,6 +3,21 @@ set -e -u
 
 rm -rf spire spire.zip src/resources src/__pycache__ resources/__pycache__
 cp -RT resources src/resources
+
+if [[ -v VERSION ]]
+then
+    echo "$VERSION" >src/resources/DEB_VERSION
+else
+    echo "Spire was not built as part of a Debian package." >src/resources/DEB_VERSION
```

Comment:
Probably worth having a less verbose string would be good?

This is long:

    Debian package version: Spire was not built as part of a Debian package.

Compared to something like this:

    Debian package version: not a debian build

Reply:
Will shorten to what you have there.